wallet-reference-server/src/main/kotlin/org/stellar/reference/wallet/WalletServerClient.kt

```diff
@@ -42,14 +43,25 @@ class WalletServerClient(val endpoint: Url = Url("http://localhost:8092")) {
         }
       }
 
-    // Parse the JSON string into a list of Person objects
-    return gson.fromJson(
-      response.body<String>(),
-      object : TypeToken<List<Sep24GetTransactionResponse>>() {}.type
-    )
+    return gson.fromJson(response.body<String>(), object : TypeToken<List<String>>() {}.type)
   }
 
-  suspend fun getLatestCallback(): Sep24GetTransactionResponse? {
+  suspend fun pollCallbacks(txnId: String?, expected: Int): List<String> {
+    var retries = 5
+    var callbacks: List<String> = listOf()
+    while (retries > 0) {
+      // TODO: remove when callbacks are de-duped
```

Comment:
Where and when the callbacks should be de-duped?

Reply:
I'm not sure.... Apparently this was fixed in the `develop` branch. When I pull that in, I'll remove this TODO and see if everything works.